### ISSUE TITLE
The play store console stated that am using FOREGROUND_SERVICE_DATA_SYNC even though I don't.

### ISSUE THREAD

**Issue by AmirDotDev0** [hannesa2/paho.mqtt.android#600]
The console is forcing me to declare tasks and provide videos for those tasks that use foreground services.
My app does not use any foreground services and never has, am currently using (implementation 'com.github.hannesa2:paho.mqtt.android:4.2') and am wondering if the new version "4.2.4" removed the foreground service.

I just want to clarify that I didn't run the MQTT client in foreground.

**Comment by hannesa2** [comment]
Here is the pull request to remote this #583 It's a leftover and needed in version 3.x

**Comment by AmirDotDev0** [comment]
Great, now after updating the dependency from "4.2.4" to version "4.3.beta1" (implementation 'com.github.hannesa2:paho.mqtt.android:4.3.beta1') the FOREGROUND_SERVICE_DATA_SYNC permission error is gone when releasing my app:

> Your app uses the FOREGROUND_SERVICE_DATA_SYNC permission. You can only use this permission if your app performs tasks noticeable to the user when they're not directly interacting with your app.

now I'm able to release my app without issues, thanks for the help.

**Comment by hannesa2** [comment]
Glad to help you. I hope it works ! Please keep me informed

**Comment by AmirDotDev0** [comment]
> FOREGROUND_SERVICE_DATA_SYNC

It worked perfectly, I republished the app, and the FOREGROUND_SERVICE_DATA_SYNC permission error is gone

**Comment by kashyap-5paisa** [comment]
Thank you very much. Really appreciated. 👍

**Comment by hannesa2** [comment]
> It worked perfectly, I republished the app, and the FOREGROUND_SERVICE_DATA_SYNC permission error is gone

Sure, publish works now.
I mean, does the MQTT functionality works properly ?

**Comment by AmirDotDev0** [comment]
> Sure, publish works now. I mean, does the MQTT functionality works properly ?

I wouldn't have published the app if it didn't work, or if there is a bug or an error, I tested the app carefully before I published it and everything seems to work fine without issues or crashes.

I've recently added a chat feature to my app and I'm using MQTT for that, and after I updated the client dependency to "4.3.beta1" I've tested the following:

1. Connect to the MQTT broker: SUCCESS
2. Subscribe to topics: SUCCESS
3. Publish messages: SUCCESS
4. Receive messages: SUCCESS

So far everything is working without problems but my testing is not done yet, I need to do more testing regarding reconnecting and disconnecting.
